tests/rbd/rbd_utils.py

```diff
@@ -490,17 +493,23 @@ def unprotect_snapshot(self, snap_name, **kw):
             cmd = f"rbd snap unprotect {snap_name}"
         return self.exec_cmd(cmd=cmd)
 
-    def image_exists(self, pool_name, image_name):
+    def image_exists(self, pool_name, image_name, **kw):
         """
         Verify if image exists in the specified pool
         Args:
             pool_name: name of the pool
             image_name: name of the image
-
+            kw: name of the namespace to check for image exist or not
         Returns:
 
         """
-        out = self.exec_cmd(cmd=f"rbd ls {pool_name} --format json", output=True)
+        if kw.get("namespace"):
```

Comment:
This also could be optimized in a similar way

```
cmd = f"rbd --pool {pool_name} ls --format json"
if kw.get("namespace"):
    cmd += f" --namespace {kw['namespace']}"
```

Reply:
Done.